### DISCUSSION TITLE
Global Deps Empty With PNPM

### DISCUSSION THREAD

**Discussion by Oliver-Turp** [raineorshine/npm-check-updates#1481] [Q&A]
I've been using ncu fine for ages with npm. I recently switch to pnpm, but `ncu -g` doesn't recognise my pnpm directory:
```
PS D:\...web-dashboard> pnpm list -g --depth=0
Legend: production dependency, optional only, dev only

C:\...\5

dependencies:
concurrently 9.1.0
nodemon 3.1.9
npm-check-updates 17.1.11
PS D:\...web-dashboard> ncu -g

No dependencies.
```
Is there a way to make that work with pnpm? I tried the in-built `pnpm outdated` but that isn't working as expected:
```
PS D:\...web-dashboard> pnpm outdated
PS D:\...web-dashboard> pnpm outdated --long
PS D:\...web-dashboard> ncu
Using pnpm
Checking D:\...\web-dashboard\package.json
[====================] 14/14 100%

 @playwright/test  ^1.49.0  →  ^1.49.1
 mongoose           ^8.8.4  →   ^8.9.0
 next               15.0.4  →   15.1.0

Run ncu -u to upgrade package.json
```

**Comment by raineorshine** [comment]
pnpm must have a different way to store global npm modules, so unfortunately that is not supported by npm-check-updates.

**Reply by Oliver-Turp** [reply]
That's a shame but thank you for replying so quickly!

Assuming pnpm continues its rise of popularity, would you consider adding wider support for it with ncu?

**Reply by raineorshine** [reply]
That would most likely have to come through community contributions. I'm personally working on other things. I'm happy to support PR's though.
